src/elements/fields/TextField.jsx

```diff
@@ -80,17 +148,20 @@ function TextField({
   applyStyles,
   fieldLabel,
   required = false,
-  fieldValue = '',
   onBlur = () => {},
-  onChange = () => {},
   onClick = () => {},
   elementProps = {},
   inlineError,
-  inputRef,
-  ...fieldProps
+  rawValue = '',
+  onAccept = () => {}
 }) {
   const servar = element.servar;
-  const inputType = fieldProps.as === 'textarea' ? 'textarea' : 'input';
+  const maskProps = {
```

Comment:
reason for spreading and then wrapping in object? function call should be enough right?

Reply:
yes you're right